data/json/items/ranged/archery.json

```diff
@@ -569,9 +569,9 @@
     "symbol": "(",
     "color": "yellow",
     "name": { "str": "composite bow (light)", "str_pl": "composite bows (light)" },
-    "description": "A traditionally-constructed bow made from wood, horn and sinew.  It has a moderate draw weight.",
+    "description": "A traditionally-constructed bow made from wood, bone, and sinew.  It has a moderate draw weight.",
```

Comment:
Sources for bone please, I've dig up a LOT of sources and none have bone as a meaningful composite material.

Reply:
My sources point toward horn, not bone.